### PR TITLE
Added support for Ubuntu 20.04 vm image

### PR DESCRIPTION
**Pull request by demodh**
## Description

This enables cluster operations with Ubuntu 20.04 vm image using AZ powershell.

## Checklist

- [yes ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ yes] The title of the PR is clear and informative
- [ yes] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ yes] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ yes] If applicable, the changes made in the PR have proper test coverage
- [ N/A] For public API changes to cmdlets:
    - [ N/A] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ N/A] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ServiceFabric/ServiceFabric/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 3.0.2
```
Comment (BethanyZhou) [inline src/ServiceFabric/ServiceFabric/ChangeLog.md:22]: Please remove new version 3.0.2 and put all change logs in the upcoming release section

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/ServiceFabric/ServiceFabric/ChangeLog.md (hunk 1 above)
Comment (demodh) [reply src/ServiceFabric/ServiceFabric/ChangeLog.md]: updated

**Review by a-santamaria — APPROVED** [review]

File: src/ServiceFabric/ServiceFabric/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
```
Comment (BethanyZhou) [inline src/ServiceFabric/ServiceFabric/ChangeLog.md:21]:
```suggestion
```

**Review by BethanyZhou — APPROVED** [review]
Looks good to me

**Pull request merged**